src/Simulation/Simulators/ResourcesEstimator/ResourcesEstimator.cs

```diff
@@ -121,24 +121,32 @@ public virtual DataTable Data
                     if (l is ICallGraphStatistics collector)
                     {
                         var results = collector.Results.ToTable();
-                        Debug.Assert(results.rows.Count() == 1);
+                        //Debug.Assert(results.rows.Count() == 1);
```

Comment:
Is it safe to delete this line completely. Might be confusing if it stays commented.

Reply:
I believe the assert is removed since for each tracer run we would add a row here - so far it seems like it wasn't intended to be used more than once.

Reply:
Thanks. Shouldn't the line better be removed completely then?

Reply:
Removed the assert